lib/recurly.js

```diff
@@ -289,6 +291,33 @@ export class Recurly extends Emitter {
       this.emit('ready');
     }
   }
+
+  /**
+   * Reports an event to the Reporter if it is available
+   *
+   * @private
+   */
+  report (...args) {
+    if (!this.reporter) return;
+    this.reporter.send(...args);
+  }
+
+  /**
+   * Binds important events to the EventDispatcher
+   *
+   * @private
+   */
+  bindReporting () {
+    if (!this.isParent) return;
+    ['focus', 'blur'].forEach(eventName => {
+      this.on(`hostedField:${eventName}`, ({ type }) => {
+        const state = this.hostedFields.state[type];
+        let meta = pick(state, ['valid', 'empty']);
```

Comment:
I think `meta` should be a `const`, as you are later adding an attribute to the object, but the reference object is the same.

Reply:
While it's true that a const can have its properties mutated, I don't think they should be.

Reply:
Ah, so you assign it with `let` to let people know attributes will be added/deleted. Got it.